enterprise/ha/src/main/java/org/neo4j/kernel/ha/BranchedDataPolicy.java

```diff
@@ -56,92 +61,9 @@ public void handle( File storeDir ) throws IOException
                 @Override
                 public void handle( File storeDir ) throws IOException
                 {
-                    for ( File file : relevantDbFiles( storeDir ) )
-                    {
-                        FileUtils.deleteRecursively( file );
-                    }
+                    cleanStoreDir( storeDir );
```

Comment:
`keep_last` deletes the entire directory. `cleanStoreDir`deletes only the "relevant" files inside that directory.